executable/dex.ksy

```diff
@@ -352,6 +306,16 @@ types:
           This offset, if non-zero, should be in the data section, and the data
           there should be in the format specified by "type_list" below.
           Additionally, there should be no reference to the type void in the list.
+    instances:
+      shorty_desc:
+        value: _root.string_ids[shorty_idx].value.data
+      params_types:
+        io: _root._io
+        pos: parameters_off
+        type: type_list
+        if: parameters_off != 0
+      return_type:
+        value: _root.type_ids[return_type_idx].type_name
```

Comment:
It would be really cool to add some documentation lines on what these are and how they would be useful for end-user.

Reply:
I have nothing against it, of course, but I didn't include any further explanations since they are analogous to `type_name` in `type_id_item`. The DEX file format uses indices into the string table and this just exports the string themselves. They are not new fields added to the format, but rather the same information displayed differently. Wouldn't adding docstrings for the instances just duplicate the information in the corresponding seq elements?